Source/Core/AudioCommon/CMakeLists.txt

```diff
@@ -33,8 +33,17 @@ endif(PULSEAUDIO_FOUND)
 
 if(WIN32)
 	set(SRCS ${SRCS} XAudio2Stream.cpp)
+
+	add_dolphin_library(audiocommon_xaudio27 "XAudio2_7Stream.cpp" "${LIBS}")
```

Comment:
This doesn't need to be a separate library, does it? In Visual Studio it's part of the AudioCommon project.